plugins/events-backend-module-bitbucket-server/README.md

```diff
@@ -0,0 +1,43 @@
+# events-backend-module-bitbucket-server
+
+Welcome to the `events-backend-module-bitbucket-server` backend plugin!
+
+This plugin is a module for the `events-backend` backend plugin
+and extends it with an `BitbucketServerEventRouter`.
+
+The event router will subscribe to the topic `bitbucketServer`
```

Comment:
Would recommend adding some docs about the proper `app-config.yaml` needed to setup this topic. I wasn't sure at first following these docs, as I thought possibly the `BitbucketServerEventRouter` was going to do that for me.

Reply:
I made a change to the bitbucket server discovery docs to describe how to add the event support. I followed the exact same format that Bitbucket Cloud does in its event docs. Do you think I should still include this same information in the individual plugin docs?

Reply:
I think it's nice to have a checklist of things to do all in one page without having to jump between a bunch of separate markdowns. I guess this is a pre-existing problem, but would be nice for things to be a bit more consolidated (maybe with a little duplication).

Reply:
Sure, I can definitely do this.

Reply:
I updated the docs in the newest commit, let me know how it is and if I should add anything more that you thought would be helpful.